server/test/routes/article.test.js

```diff
@@ -0,0 +1,341 @@
+/* eslint-disable */
+import chai from 'chai'
+import chaiHttp from 'chai-http'
+import server from '../../index';
+import models from '../../database/models';
+import tokenService from '../../utils/services/tokenService';
+import { User } from '../../database/models';
+import mockData from './mockData';
+
+let token;
+const { expect } = chai;
+const baseUrl = '/api/articles/';
+const SIGN_UP_ROUTE = '/api/users';
+chai.use(chaiHttp);
+let slug;
+
+describe('Testing articles routes', () => {
+  
+    before((done) => {
+        chai.request(server)
+          .post(SIGN_UP_ROUTE)
+          .send({
+              user: {
+                firstName:'jacky',
+                lastName:'chan',
+                username:'jackychan',
+                email: 'jackychan11@testmail.com',
+                password: '1qQw@123',
+              }
+          })
+          .end((err, res) => {
+              console.log('Some error');
+              console.log(err);
+              console.log(res);
```

Comment:
Also, check and clear out all unnecessary console.log

Reply:
done